config/initializers/devise.rb

```diff
@@ -240,7 +240,12 @@ def skip_format?
   config.maximum_attempts = 5
 
   # Time interval to unlock the account if :time is enabled as unlock_strategy.
-  config.unlock_in = 2.hours
+  time = ENV.fetch('UNLOCK_IN', 2.hours)
```

Comment:
Would be cleaner to coerce the value to an Integer with `.to_i` at the end on one line.

Reply:
It needs to be a duration though, because Devise runs [ago](https://apidock.com/rails/ActiveSupport/Duration/ago) on it